packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -0,0 +1,167 @@
+              />
+              <h3
+                className="mt-2 font-bold text-theme-label-primary typo-title3"
+                data-testid="changelogTitle"
+              >
+                {post.title}
+              </h3>
+              <time
+                className="text-theme-label-quaternary typo-callout"
+                dateTime={post.createdAt}
+                data-testid="changelogDate"
+              >
+                {postDateFormat(post.createdAt)}
+              </time>
+              <div className="mt-2 w text-theme-label-tertiary typo-callout">
+                {post.summary}
+              </div>
+            </section>
+            <footer className="flex gap-3 items-center py-3 px-4 w-full h-16 border-t border-theme-divider-tertiary">
+              <Button
+                className="btn-tertiary"
+                onClick={dismissChangelog}
+                tag="a"
+                href={post.permalink}
+                data-testid="changelogReleaseNotesBtn"
+              >
+                Release notes
+              </Button>
+              {isExtension && (
+                <Button
+                  tag={isFirefoxExtension ? 'a' : undefined}
+                  href={
+                    isFirefoxExtension
+                      ? 'https://r.daily.dev/firefoxupdate'
+                      : undefined
+                  }
+                  className="bg-cabbage-40 btn-primary"
+                  data-testid="changelogExtensionBtn"
+                  onClick={async () => {
+                    if (isFirefoxExtension) {
+                      return;
+                    }
+
+                    if (isExtension) {
+                      const sendMessage =
+                        globalThis?.browser?.runtime?.sendMessage;
+
+                      if (typeof sendMessage === 'function') {
+                        const updateResponse: { status: string } =
+                          await sendMessage({
+                            type: ExtensionMessageType.RequestUpdate,
+                          });
+
+                        let toastMessage: string;
+
+                        switch (updateResponse.status) {
+                          case 'throttled':
+                            toastMessage =
+                              'There is no update available, try again later';
+                            break;
+                          case 'no_update':
+                            toastMessage =
+                              'You are already on the latest available version';
+                            break;
+                          case 'update_available':
+                            toastMessage = 'Browser extension updated';
+                            break;
+                          default:
+                            break;
+                        }
+
+                        if (toastMessage) {
+                          toast.displayToast(toastMessage);
+                        }
```

Comment:
@idoshamun added toast as a response to "update extension" button click, just checking if it looks good to you?

Reply:
`update_available` case will not currently even be shown because when `background` script triggers `runtime.reload` the extension will just close up in the current tab and the user will get the new version when they open next tab.